### PR TITLE
Selenium -3 changes

### PR DESCRIPTION
**Pull request by RutvikChandla**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/local.log
```
Comment (Archish27) [inline .gitignore:1]: eol fixes

File: README.md [hunk 2]
```diff
@@ -1 +1,57 @@
...
+# change capabilities
+caps = Selenium::WebDriver::Remote::Capabilities.new
+caps['device'] = 'iPhone 11'
+caps['realMobile'] = 'true'
+caps['os_version'] = '14.0'
+caps['javascriptEnabled'] = 'true'
+caps['name'] = 'BStack-[Ruby] Sample Test' # test name
+caps['build'] = 'BStack Build Number 1' # CI/CD job or build name
+driver = Selenium::WebDriver.for(:remote,
+  :url =>  "https://USER_NAME:ACCESS_KEY@hub-cloud.browserstack.com/wd/hub", # Use your browserstack username and accesskey
```
Comment (Archish27) [inline README.md:34]: can you make it configurable via env variable?

Comment (RutvikChandla) [reply README.md:34]: Sure

**Comment by RutvikChandla** [comment]
@Archish27 Changes are done, please review.

File: scripts/local.rb [hunk 3]
```diff
@@ -0,0 +1,52 @@
+require 'rubygems'
+require 'selenium-webdriver'
+require "browserstack/local"
+
+# Creates an instance of Local
+bs_local = BrowserStack::Local.new
+
+# You can also set an environment variable - "BROWSERSTACK_ACCESS_KEY".
+bs_local_args = { "key" => "ACCESS_KEY" }
```
Comment (Archish27) [inline scripts/local.rb:9]: Can you correct this should be from env?

File: scripts/local.rb [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+# Starts the Local instance with the required arguments
+bs_local.start(bs_local_args)
+
+# Check if BrowserStack local instance is running
+puts bs_local.isRunning
```
Comment (Archish27) [inline scripts/local.rb:15]: Can we put appropriate log? 
eg . `Local Running: true/false`?

**Comment by RutvikChandla** [comment]
@Archish27 please review.

**Review by Archish27 — APPROVED** [review]

**Pull request merged**